### PR TITLE
Adds the Vatra M38 pistol

### PR DESCRIPTION
**Pull request by Skrem7**
# Document the changes in your pull request

Adds the Vatra M38 pistol to the game, an odd handgun that fires .38 special rounds (what the detective's gun fires). It supports an eight-round magazine and can currently only be ordered in the Security section of Cargo for 4000 credits, which gives three of the pistols with three spare magazines.

Pistol sprites done by biome, magazine sprites bastardized from the m1911 magazine done by me.

Default, rubber, and AP .38 magazines can be printed at the armory lathe with ballistic weaponry research (security lathe for rubber). Frost and talon .38 magazines can be printed with exotic ammo research. Bluespace .38 magazines can be printed with experimental ammo research.

Side note, tech shells were moved back to experimental ammo, since there's stuff to occupy exotic ammo now.

Following stats for the new .38 rounds:

- .38 armor-piercing: 18 brute, 15 AP (compared to 21/-30)
- .38 frost: -45 AP (compared to -30). If the bullet isn't blocked, chills the target. (Basically Iceblax but worse; I wanted to bring it back but not as egregiously obnoxious)
- .38 talon: 12 brute, 18 stamina (compared to 21/0, or 7/30 of rubber), sharp_edged. Causes a guaranteed open cut on a limb if damage dealt is above 7. SIDE NOTE: I did testing on fully stripped individuals: this bullet could never cause anything above the guaranteed moderate wound, so it can't be used to decapitate unless someone is super weak to wounds for some reason. If it's a concern, the wound_bonus can be reeled back in.
- .38 bluespace: 18 brute (compared to 21), speed of 0.2 (compared to 0.4) ((very very fast))

# Spriting
Pistol
image
image

Standard Magazine
image

Rubber
image

AP
image

Frost
image

Talon
image

Bluespace
image

# Wiki Documentation

Gun and ammo types to go onto Guide to Combat
Crate to go onto List of Crates

# Changelog

:cl:  Skrem_7, ToasterBiome
rscadd: Adds the Vatra M38 pistol and a respective crate in the Security section of Cargo
rscadd: Adds .38 AP, .38 frost, .38 talon, and .38 bluespace magazines
tweak: Techshell moved back to experimental ammo
bugfix: Armory WT (not actually in the armory) should be able to load other WT magazines now
imageadd: Sprites for the gun + magazine types
/:cl:

### CODE REVIEW

**Comment by ToasterBiome** [comment]
this dude did not just misspell my manufacture brand

**Comment by Skrem7** [comment]
> this dude did not just misspell my manufacture brand

I GOT IT STUCK IN MY HEAD OKAY

**Comment by RG4ORDR** [comment]
Oh god it's a fucking HiPoint

**Comment by Skrem7** [comment]
gonna re-open when i finish up a lot more work because there's more i can do now!

**Comment by MajManatee** [comment]
biome busy spriting this instead of finishing my pr

File: code/modules/projectiles/projectile/bullets/revolver.dm [hunk 1]
```diff
@@ -30,6 +30,43 @@
...
+
+/obj/item/projectile/bullet/c38/talon
+	name = ".38 talon bullet"
+	damage = 12 //Tested on naked felinids, could never cause a wound type above open cut.
+	stamina = 18
+	sharpness = SHARP_EDGED
+
+/obj/item/projectile/bullet/c38/talon/on_hit(atom/target, blocked = 0)
+	if(blocked != 100)
+		if(ishuman(target) && damage > 7)
```
Comment (Moltijoe) [inline code/modules/projectiles/projectile/bullets/revolver.dm]: please make the damage breakpoint it's own variable

**Review by Moltijoe — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/projectile/bullets/revolver.dm (hunk 1 above)
Comment (Skrem7) [reply code/modules/projectiles/projectile/bullets/revolver.dm]: its*
(yeah sure give me a moment)

**Review by cuackles — APPROVED** [review]

**Review by Moltijoe — APPROVED** [review]

**Pull request merged**